examples/tasks/psutil-file.yaml

```diff
@@ -0,0 +1,16 @@
+--- # The Header starts after this line
+  version: 1        # All tasks, like plugins, have versioning
+  schedule:         # How frequently should we run this task?
+    type: "simple"  # Simple means run forever at the interval below
+    interval: "1s"  # This accepts seconds ('s') and milliseconds  ('ms')
+  workflow:           # For a small number of statistics, you can go as small
+    collect:          # as 500ms. For a large number, more than 1s may be needed
```

Comment:
I would need to check or get someone else's feedback, but I don't know that 500ms is really the smallest. I think it's just that the current cache limit is 500ms.